torch/csrc/jit/api/compilation_unit.h

```diff
@@ -83,6 +84,16 @@ struct TORCH_API CompilationUnit {
     return true;
   }
 
+  // Version of define but with support for properties.
+  std::vector<Function*> define(
```

Comment:
Done. Although there are a bunch of callsites now where we compile free functions but still have to add empty `{}` for properties/property resolvers. I thought of adding them to the end of the argument list so as to avoid modifying all those callsites but it didn't seem as clean.